packages/playground/apps/default/preset.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AttachmentService,
+  EdgelessPreset,
+  PagePreset,
+} from '@blocksuite/blocks';
+
+const params = new URLSearchParams(location.search);
+
+class CustomAttachmentService extends AttachmentService {
+  override mounted(): void {
+    const userType = params.get('userType');
+    if (userType === 'pro') {
+      this.maxFileSize = 100 * 1000 * 1000; // 100MB
+    } else {
+      this.maxFileSize = 10 * 1000 * 1000; // 10MB
+    }
+  }
+}
+
+export const getPlaygroundPresets = () => {
```

Comment:
Just saying that using `export function getPlaygroundPresets() {}` should be more preferable in our codebase.